ref/general/microshed.adoc

```diff
@@ -1,23 +1,23 @@
 :page-layout: general-reference
 :page-type: general
-:page-description: MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
+:page-description: MicroShed testing helps you to write integration tests using Testcontainers framework for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 :page-categories: MicroShed testing
 :seo-title: Testing in a container with MicroShed testing
-:seo-description:  MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
+:seo-description:  MicroShed testing helps you to write integration tests using Testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 = Testing microservice applications
 
-Microshed testing is a Java library that helps you write true-to-production integration tests for your microservice applications using link:https://openliberty.io/blog/2019/03/27/integration-testing-with-testcontainers.html[testcontainers].
-link:https://microshed.org/microshed-testing/[MicroShed Testing] tests your application from outside without accessing it internally.
+MicroShed Testing is a Java library that helps you write true-to-production integration tests for your microservice applications with link:https://openliberty.io/blog/2019/03/27/integration-testing-with-testcontainers.html[Testcontainers] framework.
+link:https://microshed.org/microshed-testing/[MicroShed Testing] framework analyzes your application from outside the link:https://www.docker.com/why-docker[Docker] container without accessing the application internally.
 
 Integration tests assess microservices as a group, as compared to unit tests, which test each microservice as an individual unit.
 Thus, integration tests are more comprehensive than unit tests.
 However, integration tests can be more complicated and take longer to run.
-Microshed testing makes your work easier and faster by streamlining integration tests with testcontainers.
+MicroShed Testing framework makes your work streamlines your integration tests with Testcontainers framework for a better workflow.
 
 
 == Why integration testing
 
-You need to be sure that your applications works as well in production as it does in the development environment.
+You need to be sure that your applications work as well in production as it does in the development environment.
```

Comment:
subject/verb agreement- should be "as well in production as _they do_ in the development environment"